src/contentScripts.js

```diff
@@ -4,6 +4,8 @@
  *
  * @param {Array<String>} matchPattern
  * @param {import} contentScript
+ *
+ * @returns {(browser.contentScripts.RegisteredContentScript|null)} - a registered content script if Firefox, null if otherwise.
```

Comment:
I think currently this functions returns undefined, not null. Probably makes sense to return null explicitly in the else statement?

Reply:
Good catch thanks :) Can't wait to start using ts here

Reply:
amen